qa/tasks/notification_tests.py

```diff
@@ -17,34 +17,29 @@
 
 log = logging.getLogger(__name__)
 
+
 @contextlib.contextmanager
 def download(ctx, config):
-    """
-    Download the bucket notification tests from the git builder.
-    Remove downloaded test file upon exit.
-    The context passed in should be identical to the context
-    passed in to the main task.
-    """
     assert isinstance(config, dict)
-    log.info('Downloading bucket-notification-tests...')
+    log.info('Downloading bucket-notifications-tests...')
     testdir = teuthology.get_testdir(ctx)
+    actual_branch = 'master'
+    branch = ctx.config.get('suite_branch', actual_branch)
```

Comment:
@kapandya could you add an example in the documentation in this file above task.py with `suite_repo` and `suite_branch`.